content/practices/color-settings/color-settings-practice.html

```diff
@@ -0,0 +1,1055 @@
+
+....
+          </code>
+        </pre>
+
+        <h3 id="color-scheme-wikipedia">Color Scheme Example: Wikipedia Page</h3>
+
+        <p>The following example illustrates how Wikipedia supports the color scheme media query.  The example includes showing the "Appearance" sidebar allowing the user to choose the light or dark scheme and other rendering options for text size and column width.</p>
+
+        <table class="data">
+          <thead>
+            <tr>
+              <th>Media Query</th>
+              <th>Screen Shot</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <code>prefers-color-scheme: light</code>
+              </td>
+              <td>
+                <img style="width: 95%" src="./images/wikipedia-scheme-light.png" alt="Screen shot of wikipedia page with user preferences sidebar open.  The red circle shows the users settings radio button for light settings checked.">
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <code>prefers-color-scheme: dark</code>
+              </td>
+              <td>
+                <img style="width: 95%" src="./images/wikipedia-scheme-dark.png" alt="Screen shot of wikipedia page with user preferences sidebar open.  The red circle shows the users settings radio button for dark settings checked.">
+              </td>
+            </tr>
+          </tbody>
+        </table>
+
+        <h3>Testing Dark Color Scheme</h3>
+        <p>In  each operating system used by the target audience:</p>
+        <ol>
+          <li>Turn on the dark color theme.</li>
+          <li>
+            Verify contrast for text and components meet or exceed the contrast ratios specified by
+            <a href="https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum">WCAG 1.4.2: Contrast (Minimum)</a>.
+          </li>
+        </ol>
+      </section>
+
+      <section id="forced-colors">
+        <h2>Contrast Themes (Forced Colors)</h2>
+
+        <p>The <code>forced-colors</code> CSS media query provides a means for components to use the color preferences of people with visual impairments.   When the user chooses a contrast theme in the operating system (i.e. in Windows 11 Accessibility Settings > Contrast Theme), browsers set the <code>forced-colors</code> property to <code>active</code>.  When forced colors are active the browser overrides any author styling and applies the system color associated with the theme to each element.   Authors need to match the styling preferences of users with the semantics features of their components. For example text used in custom components for labels and instructions should use the same colors as the users contrast theme for text content.  Components that behave as links need to match the contrast theme styling for links and other interactive custom components should be based on the styling used for buttons, textbox and range controls for their features.</p>
```

Comment:
```suggestion
        <p>The <code>forced-colors</code> CSS media query provides a means for components to use the color preferences of people with visual impairments.   When the user chooses a contrast theme in the operating system (i.e. in Windows 11 Accessibility Settings &gt; Contrast Theme), browsers set the <code>forced-colors</code> property to <code>active</code>.  When forced colors are active the browser overrides any author styling and applies the system color associated with the theme to each element.   Authors need to match the styling preferences of users with the semantics features of their components. For example text used in custom components for labels and instructions should use the same colors as the users contrast theme for text content.  Components that behave as links need to match the contrast theme styling for links and other interactive custom components should be based on the styling used for buttons, textbox and range controls for their features.</p>
```

The above suggestion will fix the HTML linting. And running `npm run fix` should address the remaining CSS listing issues